crux-core/src/crux/io.clj

```diff
@@ -104,6 +104,9 @@
 
     f))
 
+(defn empty-folder? [^java.io.File f]
+  (<= 1 (count (take 2 (file-seq f)))))
```

Comment:
I would change this to something like `(empty? (.listFiles f))`.

Reply:
mm, me too. I wonder why I did it that way...